setup.cfg

```
@@ -19,7 +19,7 @@ author_email = will@thames.id.au
 maintainer = Ansible by Red Hat
 maintainer_email = info@ansible.com
 license = MIT
-license_file = LICENSE
+license_file = COPYING
```

Singular `license_file` is deprecated. Instead, use

```suggestion
license_files =
  COPYING
  licenses/*
```

Also note that if this option is omitted, it will fall back to including all the files matching well-known patterns meaning that our case may be covered by that as well.